.dockerignore

```diff
@@ -0,0 +1,6 @@
+*
+!public/
+!src/
+!package.json
+!yarn.lock
+!.yarn-cache
```

Comment:
I think we're adding newlines at the end of files. Install the `editorconfig` package for sublime text and it'll add them automatically.